### PR TITLE
docs: Hide Descendants on User Permissions

### PR DESCRIPTION
**Pull request by marination**
Docs for https://github.com/frappe/frappe/pull/12209
 Segregated **Advanced Control** section
screencapture-develop-in-8004-docs-user-manual-en-setting-up-users-and-permissions-user-permissions-2021-01-22-12_03_08

### CODE REVIEW

**Comment by marination** [comment]
spellcheck fails on the work 'Unico' 🤷🏻

File: erpnext_documentation/www/docs/user/manual/en/setting-up/users-and-permissions/user-permissions.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ To access User Permissions, go to:
 1. Select the user for which the rule has to be applied.
 2. Select the type of document to be allowed (for example "Company").
 3. Under For Value, select the specific item that you want to allow (the name of the "Company).
-4. If you check 'Is Default', the value selected in 'For Value' will be used as default for any future transactions by this user. That is, if company Unico Plastics Inc. is selected as 'For Value', this Company will be set as default for all future transactions by this user.
+4. If you check 'Is Default', the value selected in 'For Value' will be used by default for any future transactions by this user. That is, if company 'Unico Plastics Inc.' is selected as 'For Value', this Company will be set as default for all future transactions by this user.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/setting-up/users-and-permissions/user-permissions.md:25]:
```suggestion
4. If you check 'Is Default', the value selected in 'For Value' will be used by default for any future transactions by this user. That is if company 'Unico Plastics Inc.' is selected as 'For Value', this Company will be set as default for all future transactions by this user.
```

File: erpnext_documentation/www/docs/user/manual/en/setting-up/users-and-permissions/user-permissions.md [hunk 2]
```diff
@@ -31,15 +31,31 @@ To access User Permissions, go to:
 ## 2. More User Permission actions
 ### 2.1 Advanced control
 
-You can optionally apply user permissions only for specific document type by setting the Document Type after unticking the Apply To All Document Types checkbox.
+In Advanced control, you can have better command over where the User Permission is applied.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/setting-up/users-and-permissions/user-permissions.md:34]:
```suggestion
In Advanced Control, you can have better command over where the User Permission is applied.
```

File: erpnext_documentation/www/docs/user/manual/en/setting-up/users-and-permissions/user-permissions.md [hunk 3]
```diff
@@ -31,15 +31,31 @@ To access User Permissions, go to:
 ## 2. More User Permission actions
 ### 2.1 Advanced control
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/setting-up/users-and-permissions/user-permissions.md:32]:
```suggestion
### 2.1 Advanced Control
```

File: erpnext_documentation/www/docs/user/manual/en/setting-up/users-and-permissions/user-permissions.md [hunk 4]
```diff
@@ -31,15 +31,31 @@ To access User Permissions, go to:
...
 
 In the above User Permission, the user will be able to access only Sales Orders of the selected company.
 
 **Note:** If **Applicable For** is not set, User Permission will apply across all related Document Types.
 
+### 2.1.2. Hide Descendants
+
+The value of **Allow** could be a DocType with a Tree View, which will have records with a parent-child or ancestor-descendant relationship.
+
+Let's assume **For Value**, 'Unico Plastics Inc.', has a child company 'Unico Toys'. When a User Permission is created for 'Unico Plastics Inc.', permissions for it's descendants are granted as well.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/setting-up/users-and-permissions/user-permissions.md:51]:
```suggestion
Let's assume **For Value**, 'Unico Plastics Inc.', has a child company 'Unico Toys'. When a User Permission is created for 'Unico Plastics Inc.', permissions for its descendants are granted as well.
```

File: erpnext_documentation/www/docs/user/manual/en/setting-up/users-and-permissions/user-permissions.md [hunk 5]
```diff
@@ -65,7 +81,7 @@ Ticking on the Show Permissions checkbox will show the read/write/submit and oth
 
 Note: If you cannot access Sales Order or any other document type in this list, make sure you've set the [roles](/docs/user/manual/en/setting-up/users-and-permissions/role-based-permissions) correctly.
 
-For example, User Bruce is restricted to Company Unico Plastics Inc.
+For example, User Bruce is restricted to Company 'Unico Plastics Inc.'
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/setting-up/users-and-permissions/user-permissions.md:84]:
```suggestion
For example, the user, Bruce is restricted to Company 'Unico Plastics Inc.'
```

**Review by michellealva — CHANGES_REQUESTED** [review]

**Review by michellealva — APPROVED** [review]

**Pull request merged**